src/App.jsx

```diff
@@ -14,33 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortByParameter(goods, { param = '' }) {
+  let goodsArr = goods;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (param) {
+    case 'alphabetically': {
```

Comment:
move `alphabetically ` and `length ` to an `enum`
```
enum SortType {
  ALPHABETICALLY = 'alphabetically',
  LENGTH = 'length',
  DEFAULT = '',
}
```

Reply:
It's JSX not typescript